lib/cloudantBaseService.ts

```diff
@@ -209,4 +215,17 @@ export default abstract class CloudantBaseService extends BaseService {
     }
     return super.createRequest(parameters);
   }
+
+  private static setAgent(options: UserOptions) {
+    const agentOptions = {
```

Comment:
Not essential, but we could be more clear about exactly what these options are.
```suggestion
    const cloudantDefaultAgentOptions = {
```

Reply:
4b7bce2 and 59d4f78